dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -156,6 +159,9 @@ deploy_che_to_ocp() {
     docker run -v "${CONFIG_DIR}":/to_remove alpine sh -c "rm -rf /to_remove/" || true
     docker run -t --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${CONFIG_DIR}":/data -e IMAGE_INIT="$IMAGE_INIT" -e CHE_MULTIUSER="$CHE_MULTIUSER" eclipse/che-cli:${CHE_IMAGE_TAG} config --skip:pull --skip:nightly
     cd "${CONFIG_DIR}/instance/config/openshift/scripts/"
+    if $CHE_REMOVE_PROJECT; then
+      remove_che_from_ocp
```

Comment:
I don;t think that remove che from ocp should be there

Reply:
Needs to be there the way in the way I have programmed. The way that cli parameters are currently parsed I cannot be sure that remove will run before deploy if user puts `--deploy-che` before `--remove-che` in the cli command. This makes sure that remove is run before deploy no matter the order of the parameters. There is a way to search parameters in order but I tried not to change the existing code too much.

Reply:
that could be invoked right in `parse_args()` instead of setting VAR just call the method.

Reply:
That is the issue though. If deploy is executed before removal it will fail. `parse_args()` does not ordering parameters so I cannot be sure the removal function will run before deploy function.

Reply:
I meant just change this:

```
parse_args()
...

 +    if [[ "$@" == *"--remove-che"* ]]; then
 +      CHE_REMOVE_PROJECT=true
 +    fi
```

to this:

```
parse_args()
...

 +    if [[ "$@" == *"--remove-che"* ]]; then
 +      remove_che_from_ocp
 +    fi
```

it will be executed before `case`

Reply:
@riuvshin Thanks. Yes that makes sense. Didn't think about doing it that way :) . Fixed please review.